src/pages/UsersPage.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components/macro'
+import User from '../components/User'
+
+const data = require('../data/data.json')
+
+const users = data
+
+export default function UsersPage() {
+  return (
+    <Wrapper>
+      {users.map(({ name, phone, email, department }) => (
+        <SingleEntry key={name}>
+          <User
```

Comment:
You pass props here but you don't receive them in your `User`-component
like `export default function User({name, phone, mail, department, skills})`